shared/references/cce-redhat-avail.txt

```diff
@@ -4921,3 +4920,530 @@ CCE-90791-5
 CCE-90792-3
 CCE-90793-1
 CCE-90794-9
+CCE-83448-1
+CCE-83449-9
```

Comment:
I think this bunch of added CCEs should not be here.
I was added in 55817077aa4c5446b83a3453f04a6b8438cc4b1e